packages/@react-spectrum/s2/stories/TableView.stories.tsx

```diff
@@ -160,6 +249,22 @@ export const Dynamic = {
   }
 };
 
+export const DynamicCustomMenus = {
+  render: DynamicTableWithCustomMenus,
+  args: {
+    ...Example.args,
+    disabledKeys: ['Foo 5']
+  }
+};
+
+export const DynamicSortableCustomMenus = {
+  render: DynamicSortableTableWithCustomMenus,
+  args: {
+    ...Example.args,
+    disabledKeys: ['Foo 5']
+  }
+};
+
```

Comment:
We should look at potentially combining some of the stories since these are also our docs